xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.h

```diff
@@ -132,6 +141,8 @@ class CProcessorHD : public ID3DResource
   DXGI_COLOR_SPACE_TYPE GetDXGIColorSpaceTarget(const DXGIColorSpaceArgs& csArgs,
                                                 bool supportHDR,
                                                 bool limitedRange) const;
+  static DXGI_COLOR_SPACE_TYPE AvToDxgiColorSpace(const DXGIColorSpaceArgs& csArgs);
+  std::vector<DXGI_FORMAT> ProcessorOutputFormats() const;
```

Comment:
Maybe rename to `GetProcessorOutputFormats`

Reply:
good idea